### ISSUE TITLE
form-verifiers doesn't correctly clear errors from checkboxes in a group

### ISSUE THREAD

**Issue by cvializ** [ampproject/amphtml#11905]
... more details coming soon

**Comment by dvoytenko** [comment]
@cvializ pls triage

**Comment by aghassemi** [comment]
@cvializ please add details or close. Thanks.

**Comment by cvializ** [comment]
If I remember correctly, this was the result of validation messages only appearing for the first checkbox with the same name as others. I have since discovered this is an expected behavior in the HTML spec and should be closed
